src/Files.App/Contexts/ContentPage/IContentPageContext.cs

```diff
@@ -32,6 +32,9 @@ public interface IContentPageContext : INotifyPropertyChanged
 
 		bool ShowSearchUnindexedItemsMessage { get; }
 
+		bool IsGitRepository { get; }
```

Comment:
Each property should have a new line break.

```suggestion
		bool IsGitRepository { get; }

```

Reply:
Shouldn't we group together related properties?

Reply:
In this context it's probably best to group them.